packages/main/src/StepInput.js

```diff
@@ -0,0 +1,540 @@
+		 *
+		 * @type {string}
+		 * @defaultvalue "left"
+		 * @public
+		 */
+		textAlign: {
+			type: String,
+			defaultValue: "left",
+		},
+
+		/**
+		 * Determines the number of digits after the decimal point of the <code>ui5-step-input</code>.
+		 *
+		 * @type {Integer}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		valuePrecision: {
+			type: Integer,
+			defaultValue: 0,
+		},
+
+		/**
+		 * Defines the aria-label attribute for the <code>ui5-step-input</code>.
+		 *
+		 * @type {String}
+		 * @since 1.0.0-rc.9
+		 * @private
+		 * @defaultvalue ""
+		 */
+		ariaLabel: {
+			type: String,
+		},
+
+		/**
+		 * Receives id(or many ids) of the elements that label the <code>ui5-step-input</code>.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 * @since 1.0.0-rc.9
+		 */
+		ariaLabelledby: {
+			type: String,
+			defaultValue: "",
+		},
+
+		_decIconDisabled: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_incIconDisabled: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_focused: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_inputFocused: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_previousValue: {
+			type: Float,
+			noAttribute: true,
+		},
+
+	},
+	slots: /** @lends sap.ui.webcomponents.main.StepInput.prototype */ {
+		/**
+		 * Defines the value state message that will be displayed as pop up under the <code>ui5-step-input</code>.
+		 * <br><br>
+		 *
+		 * <b>Note:</b> If not specified, a default text (in the respective language) will be displayed.
+		 * <br>
+		 * <b>Note:</b> The <code>valueStateMessage</code> would be displayed,
+		 * when the <code>ui5-step-input</code> is in <code>Information</code>, <code>Warning</code> or <code>Error</code> value state.
+		 * @type {HTMLElement}
+		 * @since 1.0.0-rc.7
+		 * @slot
+		 * @public
+		 */
+		valueStateMessage: {
+			type: HTMLElement,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.StepInput.prototype */ {
+		/**
+		 * Fired when the input operation has finished by pressing Enter or on focusout.
+		 *
+		 * @event
+		 * @public
+		*/
+		change: {},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ *
+ * <h3>Usage</h3>
+ *
+ * For the <code>ui5-step-input</code>
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import @ui5/webcomponents/dist/StepInput.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.StepInput
+ * @extends UI5Element
+ * @tagname ui5-step-input
+ * @public
+ */
+class StepInput extends UI5Element {
+
+	constructor() {
+		super();
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
+	}
+
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return StepInputCss;
+	}
+
+	static get template() {
+		return StepInputTemplate;
+	}
+
+	static get dependencies() {
+		return [];
+	}
+
+	static async onDefine() {
+		await Promise.resolve([
```

Comment:
If it is one thing to await , you can just write it:

```js
static async onDefine() {
await fetchI18nBundle("@ui5/webcomponents"),
}```

Reply:
Done